### PR TITLE
SUBMARINE-647. Change Workbench user name

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?
The user name at the top-right corner is liuxun, should change to admin

### What type of PR is it?
[Bug Fix]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-647

### How should this be tested?
https://travis-ci.org/github/kobe860219/submarine/builds/731279516

### Screenshots (if appropriate)
截圖 2020-09-29 下午9 54 31

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.html [hunk 1]
```diff
@@ -71,7 +71,7 @@ <h1>Submarine</h1>
           class="operation-user-info"
         >
           <nz-avatar nzIcon="user" nzSize="small"></nz-avatar>
-          {{ userInfo.name }}
+          admin
```
Comment (xunliu) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.html:74]: No! `userInfo.name` This username is queried from the database and cannot be written as a fixed value.

Comment (xunliu) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.html:74]: No! `userInfo.name` This username is queried from the database and cannot be written as a fixed value.

**Review by xunliu — CHANGES_REQUESTED** [review]

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.ts [hunk 2]
```diff
@@ -130,7 +130,7 @@ export class WorkbenchComponent implements OnInit {
     if (this.authService.isLoggedIn) {
       this.userInfo$ = this.userService.fetchUserInfo().pipe(
         tap((userInfo) => {
-          this.nzNotificationService.success('Welcome', `Welcome back, ${userInfo.name}`);
+          this.nzNotificationService.success('Welcome', `Welcome to submarine workbench!`);
```
Comment (xunliu) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.ts:133]: `userInfo.name` This username is queried from the database and cannot be written as a fixed value.

**Comment by aeioulisa** [comment]
https://github.com/apache/submarine/blob/cdb2e7f8b17c1badbcd0601d8200b947f9a37a97/submarine-server/server-core/src/main/java/org/apache/submarine/server/workbench/rest/SysUserRestApi.java#L228

**Review by aeioulisa — COMMENTED** [review]
https://github.com/apache/submarine/blob/cdb2e7f8b17c1badbcd0601d8200b947f9a37a97/submarine-server/server-core/src/main/java/org/apache/submarine/server/workbench/rest/SysUserRestApi.java#L228

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.html [hunk 3]
```diff
@@ -71,7 +71,7 @@ <h1>Submarine</h1>
           class="operation-user-info"
         >
           <nz-avatar nzIcon="user" nzSize="small"></nz-avatar>
-          {{ userInfo.name }}
+          {{ userInfo.name }}
```
Comment (aeioulisa) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.html:74]:
```suggestion
          {{ userInfo.name }}
```

**Review by xunliu — APPROVED** [review]
LGTM
Will merge if no more comments.

@kobe860219 In the next 0.6.0 version, we need to create a new ticket and use the data for the database to replace the mock data in the [SysUserRestApi.java:info()](https://github.com/apache/submarine/blob/master/submarine-server/server-core/src/main/java/org/apache/submarine/server/workbench/rest/SysUserRestApi.java#L162) function.

**Pull request closed without merging**